### PR TITLE
BaSD-MS-45/46: Create Input and Select Shared

### PR DESCRIPTION
**Pull request by Brunomoyano4**
### Create Input and Select Shared

- Create input shared and replace it in all components
- Create select shared and replace it in all components

### CODE REVIEW

File: src/Components/Shared/Input/input.module.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.container {
```
Comment (ravegliamartin) [inline src/Components/Shared/Input/input.module.css:1]: This class is not in use. 

Maybe, to help on having a cleaner code is better to remove it. 
Wrapping the `Input` inner elements (`label` and `input`) with a container might be a good idea as well, so it renders as one block parent element with two childs.

File: src/Components/Shared/Input/index.js [hunk 2]
```diff
@@ -1,18 +1,19 @@
 import styles from './input.module.css';
 
-function Input(props) {
-  return (
+const Input = (props) => (
+  <>
+    {!!props.label} {<label className={styles.label}>{props.label}</label>}
```
Comment (ravegliamartin) [inline src/Components/Shared/Input/index.js:5]: I think this line of code should be `{props.label && <label className={styles.label}>{props.label}</label>}` 

Right now it is rendering the `label` tag every time, but it's not visible cause when there is no `props.label` the width is equal to 0.

File: src/Components/Shared/Select/index.js [hunk 3]
```diff
@@ -1,19 +1,17 @@
 import styles from './select.module.css';
 
-const Select = ({ value, onChange, label, id, required, options }) => {
+const Select = (props) => {
   return (
     <>
-      <label className={styles.label} htmlFor={id}>
-        {label}
-      </label>
+      {!!props.label} {<label className={styles.label}>{props.label}</label>}
```
Comment (ravegliamartin) [inline src/Components/Shared/Select/index.js:6]: I think this line of code should be `{props.label && <label className={styles.label}>{props.label}</label>}` 

Right now it is rendering the `label` tag every time, but it's not visible cause when there is no `props.label` the width is equal to 0.

**Review by ravegliamartin — CHANGES_REQUESTED** [review]

File: src/Components/Shared/Input/input.module.css (hunk 1 above)
Comment (Brunomoyano4) [reply src/Components/Shared/Input/input.module.css:1]: wrapping the inputs is a good idea, but within each form already belong to another container, then we would have as a result a container within another container

File: src/Components/Postulants/Form/postulantsForm.module.css [hunk 4]
```diff
@@ -71,6 +71,11 @@
   font-weight: 600;
 }
 
+<<<<<<< HEAD:src/Components/Postulants/Form/postulantsForm.module.css
```
Comment (nicolasviola) [inline src/Components/Postulants/Form/postulantsForm.module.css:74]: Aca tenes conflictos que no fueron resueltos y comiteaste los cambios sin resolverlos. Por favor chequealo porque va a romper toda la app. Es muy importante probar todo antes de subir los cambios

File: src/Components/Shared/Input/input.module.css [hunk 5]
```diff
@@ -1,7 +1,13 @@
 .input {
   padding: 10px;
+  width: 100%;
   border-color: #72bc8eaa;
-  border-radius: 6px;
+  border-radius: 2px;
+  outline: none;
+	box-sizing: border-box;
```
Comment (nicolasviola) [inline src/Components/Shared/Input/input.module.css:7]: Corregir la indentacion por favor

**Review by nicolasviola — CHANGES_REQUESTED** [review]

File: src/Components/Shared/Input/input.module.css (hunk 5 above)
Comment (Brunomoyano4) [reply src/Components/Shared/Input/input.module.css:7]: image

Se ve que era un problema invisible porque no se me muestra en el código, espero con el formatter se solucione

**Review by nicolasviola — APPROVED** [review]

**Pull request merged**